plugins/flashrom/flashrom.quirk

```diff
@@ -32,3 +32,14 @@ VersionFormat=number
 [0ee5867c-93f0-5fb4-adf1-9d686ea1183a]
 Branch=coreboot
 VersionFormat=number
+
+# match all devices with this udev subsystem
+[I2C]
+Plugin = flashrom
+
+# Parade PS175
+[1AF80175:00]
+Plugin=flashrom
+FlashromProgrammer=lspcon_i2c_spi
+Name=PS175
+Vendor=Parade
```

Comment:
It's not clear to me  - how is this I2C device actually getting probed?  Is there some kernel driver that is exporting that it exists?  Maybe some other documentation / links in README.md would be useful

Reply:
To be honest I am not sure. Here is the sysfs device on my system and its modalias:
```
# cat /sys/devices/pci0000:00/0000:00:15.2/i2c_designware.1/i2c-7/i2c-1AF80175:00/modalias
acpi:1AF80175:
```

Reply:
OK, so it's programmatically described in the ACPI tables as being something hanging off that HID controller.  That makes somewhat more sense.

Reply:
I guess the one thing I want to make sure though, that ACPI ID - it's only ever used with Parade?  Like they've reserved that namespace.  If not, please make sure you get the firmware fixed to a namespace only for Parade.

Reply:
That id is a combo of vid pid (4 characters each) which I understand is specific to Parade.